gammapy/data/event_list.py

```diff
@@ -365,35 +366,64 @@ def select_region(self, regions, wcs=None):
         mask = geom.contains(self.radec)
         return self.select_row_subset(mask)
 
-    def select_parameter(self, parameter, band):
-        """Select events with respect to a specified parameter.
+    @deprecated_renamed_argument(["band"], ["values"], ["2.0"])
+    def select_parameter(self, parameter, values, is_range=True):
+        """
+        Event selection according to parameter values, either in a range or exact matches.
 
         Parameters
         ----------
         parameter : str
-            Parameter used for the selection. Must be present in `self.table`.
-        band : tuple or `astropy.units.Quantity`
-            Minimum and maximum value for the parameter to be selected (minimum <= parameter < maximum).
-            If parameter is not dimensionless, a Quantity is required.
+            Column name to filter on
+        values : str, `~numpy.nan`, tuple, list or `~numpy.ndarray`
+            Value(s) for the parameter to be selected on.
+        is_range : `bool`
```

Comment:
When we have an argument that is defined in the function with a default value, we also need to add `optional` to the docstring

```suggestion
        is_range : `bool`, optional
```